features/latest-documents.feature

```diff
@@ -7,7 +7,7 @@ Feature: List of most recently published documents
   Scenario: Latest documents on topical event page
     Given a topical event with published documents
     When I view that topical event page
-    Then I can see some of the latest documents
+    Then I can see some of the latest documents from rummager
```

Comment:
Could we replace the implementation of the "Then I can see some of the latest documents" step definition? "from rummager" is an implementation detail, it's not something the user is aware of.